### PR TITLE
Added an entry on treemap

### PR DESCRIPTION
**Pull request by NimitS1**
Treemaps would be really useful as we often have to sort data so I felt that it would be a good idea to provide it upfront rather than letting the user go and fetch data. This PR is open to discussion :)

### CODE REVIEW

File: java.html.markdown [hunk 1]
```diff
@@ -153,6 +154,9 @@ public class LearnJava {
         //            interface. This allows the execution time of basic
         //            operations, such as get and insert element, to remain
         //            constant even for large sets.
+        // TreeMap - This class is a sorted tree structure. It implemets a red
```
Comment (vendethiel) [inline java.html.markdown]: implements

**Comment by levibostian** [comment]
Can you merge master, please?

**Comment by vendethiel** [comment]
@NimitS1 pretty please? :)

**Comment by vendethiel** [comment]
@NimitS1 ? please!

**Comment by NimitS1** [comment]
Hey I am sorry!
I don't have laptop access till 6th July. I won't be able to work on this
till then :-/

**Comment by NimitS1** [comment]
Closing this pull request. I have updated the code in the PR: 2300

**Pull request closed without merging**